.github/workflows/func.yaml

```diff
@@ -76,13 +76,18 @@ on:
         type: string
         default: ""
         description: "External juju controller name to use"
+      pytest-env:
+        required: false
+        type: string
+        description: "Extra pytest environment for pytest, base64 encoded env file"
+      zaza-yaml:
+        required: false
+        type: string
+        description: "Extra zaza yaml, base64 encoded yaml string"
```

Comment:
nit: A nicer way to say this could be: `Base64 encoded .zaza.yaml file for multi-cloud/region configuration.`

Reply:
`zaza.yaml` is not just for multi-cloud/region configuration. You can use to change other config in the same time.